### PR TITLE
HIP-584: Fix regression issues with acceptance tests

### PR DESCRIPTION
**Pull request by IvanKavaldzhiev**
**Description**:

Fix acceptance test issues and enable `approve` and `isApprovedForAll` precompiles by default
**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by attest08** [comment]
LGTM

File: hedera-mirror-test/src/test/resources/features/contract/precompile.feature [hunk 1]
```diff
@@ -1,59 +1,59 @@
...
-        And the contract call REST API should return the expiry token info for a non fungible token
-        And the contract call REST API should return the token key for a non fungible token
-        And the contract call REST API should return the name by direct call for a non fungible token
-        And the contract call REST API should return the symbol by direct call for a non fungible token
-        And the contract call REST API should return the total supply by direct call for a non fungible token
-        And the contract call REST API should return the ownerOf by direct call for a non fungible token
-        And the contract call REST API should return the getApproved by direct call for a non fungible token
-        And the contract call REST API should return the isApprovedForAll by direct call for a non fungible token
-        And the contract call REST API should return the custom fees for a non fungible token
+  @precompile @web3 @acceptance
```
[the remark this answers is not in the document]
Comment (IvanKavaldzhiev) [reply hedera-mirror-test/src/test/resources/features/contract/precompile.feature]: Done

File: hedera-mirror-test/src/test/resources/features/contract/erc.feature [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
-        Then I approve <spenderName> for nft
-        And I call the erc contract via the mirror node REST API for token getApproved with response BOB
-        Then I approve <approvedForAllSpenderName> for nft all serials
-        And I call the erc contract via the mirror node REST API for token isApprovedForAll with response true
-        Then I approve <tokenAllowanceSpender> with <allowances>
-        And I call the erc contract via the mirror node REST API for token allowance with allowances
-        Examples:
-            | supplyType | spenderName | approvedForAllSpenderName | tokenAllowanceSpender | allowances |
-            | "INFINITE" | "BOB"       | "ALICE"                   | "ALICE"               | 2          |
+  @web3 @erc @acceptance
```
[the remark this answers is not in the document]
Comment (IvanKavaldzhiev) [reply hedera-mirror-test/src/test/resources/features/contract/erc.feature]: Done

File: hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/PrecompileContractFeature.java [hunk 3]
```diff
@@ -36,6 +42,9 @@
 import io.cucumber.java.en.And;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.*;
```
[the remark this answers is not in the document]
Comment (IvanKavaldzhiev) [reply hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/PrecompileContractFeature.java]: Done

File: hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/TokenClient.java [hunk 4]
```diff
@@ -482,4 +484,14 @@ public long getTokenBalance(AccountId accountId, TokenId tokenId) {
         log.debug("{}'s token balance is {} {} tokens", accountId, balance, tokenId);
         return balance;
     }
+
+    public void validateAddress(final String actualAddress) {
```
[the remark this answers is not in the document]
Comment (IvanKavaldzhiev) [reply hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/TokenClient.java]: Done

**Review by steven-sheehy — APPROVED** [review]

**Pull request merged**